solarforecastarbiter/io/reference_observations/sfa_reference_sites.csv

```diff
@@ -209,3 +209,18 @@ interval_length,name,latitude,longitude,elevation,network_api_id,network_api_abb
 5,Denali AK,63.45,-150.87,2225,AK_Denali_27_N,AK_Denali_27_N,America/Anchorage,"Diamond, H. J., T. R. Karl, M. A. Palecki, C. B. Baker, J. E. Bell, R. D. Leeper, D. R. Easterling, J. H. Lawrimore, T. P. Meyers, M. R. Helfert, G. Goodge, and P. W. Thorne, 2013: U.S. Climate Reference Network after one decade of operations: status and assessment. Bull. Amer. Meteor. Soc., 94, 489-498. doi: 10.1175/BAMS-D-12-00170.1",NOAA USCRN
 5,Toolik Lake AK,68.64,-149.39,2461,AK_Toolik_Lake_5_ENE,AK_Toolik_Lake_5_ENE,America/Anchorage,"Diamond, H. J., T. R. Karl, M. A. Palecki, C. B. Baker, J. E. Bell, R. D. Leeper, D. R. Easterling, J. H. Lawrimore, T. P. Meyers, M. R. Helfert, G. Goodge, and P. W. Thorne, 2013: U.S. Climate Reference Network after one decade of operations: status and assessment. Bull. Amer. Meteor. Soc., 94, 489-498. doi: 10.1175/BAMS-D-12-00170.1",NOAA USCRN
+1.0,"E11: Byron, Oklahoma",36.88100051879883,-98.28500366210938,367.5,sgpqcrad1longE11.c1,sgp,America/Chicago,,DOE ARM
+1.0,"E12: Pawhuska, Oklahoma",36.84099960327149,-96.427001953125,328.0,sgpqcrad1longE12.c1,sgp,America/Chicago,,DOE ARM
+1.0,"E33: Newkirk, Oklahoma",36.92549896240234,-97.08170318603516,356.0,sgpqcrad1longE33.c1,sgp,America/Chicago,,DOE ARM
+1.0,"E34: Maple City, Kansas",37.06940078735352,-96.7605972290039,404.0,sgpqcrad1longE34.c1,sgp,America/Chicago,,DOE ARM
+1.0,"E37: Waukomis, Oklahoma",36.31060028076172,-97.9280014038086,387.0,sgpqcrad1longE37.c1,sgp,America/Chicago,,DOE ARM
+1.0,"E9: Ashton, Kansas",37.132999420166016,-97.26599884033205,372.0,sgpqcrad1longE9.c1,sgp,America/Chicago,,DOE ARM
+1.0,"North Slope of Alaska (NSA), Barrow, Alaska",71.322998046875,-156.60899353027344,5.0,nsametC1.b1,nsa,America/Anchorage,,DOE ARM
+1.0,"Southern Great Plains (SGP), Anthony, Kansas",37.1509017944336,-98.36199951171876,417.0,sgpmetE31.b1,sgp,America/Chicago,,DOE ARM
+1.0,"Southern Great Plains (SGP), Lamont, Oklahoma",36.60499954223633,-97.48500061035156,307.0,sgpmetE13.b1,sgp,America/Chicago,,DOE ARM
+1.0,"Southern Great Plains (SGP), Maple City, Kansas",37.06940078735352,-96.7605972290039,404.0,sgpmetE34.b1,sgp,America/Chicago,,DOE ARM
+1.0,"Southern Great Plains (SGP), Marshall, Oklahoma",36.116600036621094,-97.51119995117188,326.0,sgpmetE36.b1,sgp,America/Chicago,,DOE ARM
+1.0,"Southern Great Plains (SGP), Medford, Oklahoma",36.81900024414063,-97.81990051269531,335.0,sgpmetE32.b1,sgp,America/Chicago,,DOE ARM
+1.0,"Southern Great Plains (SGP), Omega, Oklahoma",35.87969970703125,-98.17279815673828,373.0,sgpmetE38.b1,sgp,America/Chicago,,DOE ARM
+1.0,"Southern Great Plains (SGP), Ringwood, Oklahoma",36.430999755859375,-98.28399658203124,414.0,sgpmetE15.b1,sgp,America/Chicago,,DOE ARM
+1.0,"Southern Great Plains (SGP), Tryon, Oklahoma",35.86149978637695,-97.06950378417969,309.0,sgpmetE35.b1,sgp,America/Chicago,,DOE ARM
```

Comment:
Looks like attribution is hard for the ARM sites, but should we at least link to the pages to generate the proper citation like https://www.arm.gov/capabilities/vaps/qcrad?

Reply:
Yeah! I actually hadn't found that citation generator before. Thanks for taking a look for that. I'll add this url to the attribution column of qcrad sites.